WickedEngine/wiScene_Serializers.cpp

```diff
@@ -1163,6 +1163,9 @@ namespace wi::scene
 				}
 			}
 		}
+
+		// Root Bone Name
+		SerializeEntity(archive, rootMotionBone, seri);
```

Comment:
If you serialize a new property, you need to create a new version for that component. It is done in wiScene.h, where the component type is registered on the scene, the last argument can be given a version number, that must be always increased by one when serializing something new, to keep backward compatibility.

Then in the serialization code, you check if the version number is high enough before serializing the property:
```
if(seri.GetVersion() < 2) // requires version 2 of AnimationComponent
{
  // Root Bone Name
  SerializeEntity(archive, rootMotionBone, seri);
}
```